src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -139,6 +139,7 @@ private void ResetTimeoutTimer()
             _connectionActive = new CancellationTokenSource();
             using (var memoryStream = new MemoryStream())
             {
+                _logger.SendingHubNegotiate();
```

Comment:
Do we have a log after it succeeds or fails?

Reply:
If it succeeds StartAsync returns successfully, if it fails you'll get an exception from StartAsync. Would a log still be useful?

Reply:
No, that's fine.